pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -334,21 +348,20 @@ func (r *RsyncPodProgressTask) getRsyncClientContainerStatus(podRef *kapi.Pod) (
 		}
 	}
 	if containerStatus == nil {
-		if podRef.Status.Phase == kapi.PodFailed {
-			rsyncPodStatus.PodPhase = kapi.PodFailed
-			rsyncPodStatus.LogMessage = podRef.Status.Message
-			rsyncPodStatus.ContainerElapsedTime = nil
-			return &rsyncPodStatus, nil
-		}
-		return nil, fmt.Errorf("container not found")
+		log.Error(fmt.Errorf("container not found"),
+			"failed to find container in Rsync Pod on source cluster",
+			"container", RsyncContainerName, "pod", path.Join(podRef.Namespace, podRef.Name))
+		return &rsyncPodStatus
 	}
 	switch {
 	case containerStatus.Ready:
 		rsyncPodStatus.PodPhase = kapi.PodRunning
 		numberOfLogLines := int64(5)
 		logMessage, err := r.getPodLogs(podRef, RsyncContainerName, &numberOfLogLines, false)
 		if err != nil {
-			return &rsyncPodStatus, err
+			log.Error(err, "failed to get logs from Rsync Pod on source cluster",
```

Comment:
Is this something that we regularly see during retries? Just wondering if it's an error worthy of stacktracing or not (appears to stacktrace currently).

Reply:
nit: I've been capitalizing the first letter of every log message throughout the controller.

Reply:
@djwhatle this doesn't necessarily mean something is broken

Reply:
Ok, if it's expected, I think it would make sense to print it at standard info level so that the user doesn't see a stacktrace.

Reply:
@djwhatle fixed now